### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by kuwv** [libgit2/pygit2#1145] [General]
Hi @jdavid,

I see the commit signing changes haven't been released yet. You had mentioned some other work that was ongoing. Do you need help with getting the next release completed?

Reason I ask is because I'd like to get these changes integrated into one of my projects: https://pypi.org/project/proman-versioning/

**Comment by jdavid** [comment]
The problem is Windows CI is broken now, this is because of the changes in libgit2 v1.4.3
But if no one fixes it, I'll just downgrade the Windows wheels to 1.4.2 and release.

**Comment by jdavid** [comment]
@kuwv it's done
